x-pack/legacy/plugins/ml/server/models/job_validation/messages.js

```diff
@@ -252,14 +253,27 @@ export const getMessages = () => {
       }),
       url: 'https://www.elastic.co/guide/en/elasticsearch/reference/{{version}}/ml-job-resource.html#ml-job-resource'
     },
+    job_id_invalid_max_length: {
+      status: 'ERROR',
+      text: i18n.translate('xpack.ml.models.jobValidation.messages.jobIdInvalidMaxLengthErrorMessage', {
+        defaultMessage: 'Job name can\'t contain more than {maxLength, plural, one {# character} other {# characters}}.',
```

Comment:
As above, suggest rephrasing to `Job ID must be no more than 64 characters long.`